### PR TITLE
adjust job TTL seconds

### PR DESCRIPTION
**Pull request by D0m021ng**
add job TTL seconds for failed job.

### CODE REVIEW

File: pkg/common/config/config.go [hunk 1]
```diff
@@ -89,8 +89,9 @@ type FsServerConf struct {
 type ReclaimConfig struct {
 	CleanJob             bool `yaml:"isCleanJob"`
 	SkipCleanFailedJob   bool `yaml:"isSkipCleanFailedJob"`
+	FailedJobTTLSeconds  int  `yaml:"failedJobTTLSeconds,omitempty"`
 	JobTTLSeconds        int  `yaml:"jobTTLSeconds"`
-	JobPendingTTLSeconds int  `yaml:"JobPendingTTLSeconds,omitempty"`
+	JobPendingTTLSeconds int  `yaml:"jobPendingTTLSeconds,omitempty"`
 }
```
Comment (qiaoshuangshuang) [inline pkg/common/config/config.go:95]: pendingJobTTLxx

File: pkg/common/config/config.go [hunk 2]
```diff
@@ -89,8 +89,9 @@ type FsServerConf struct {
 type ReclaimConfig struct {
 	CleanJob             bool `yaml:"isCleanJob"`
 	SkipCleanFailedJob   bool `yaml:"isSkipCleanFailedJob"`
+	FailedJobTTLSeconds  int  `yaml:"failedJobTTLSeconds,omitempty"`
 	JobTTLSeconds        int  `yaml:"jobTTLSeconds"`
```
Comment (qiaoshuangshuang) [inline pkg/common/config/config.go]: 只代表success状态的作业清理时间？

File: pkg/job/runtime/kubernetes/controller/job_gc_handler.go [hunk 3]
```diff
@@ -55,6 +61,33 @@ func (j *JobGarbageCollector) update(old, new interface{}) {
...
+		log.Warnf("convert ttl second string %s to int failed, err: %v", ttlStr, err)
+	}
+	// get job TTL seconds from config
+	ttlSeconds := DefaultJobTTLSeconds
+	switch status {
+	case schema.StatusJobSucceeded:
+		if config.GlobalServerConfig.Job.Reclaim.JobTTLSeconds != 0 {
+			ttlSeconds = config.GlobalServerConfig.Job.Reclaim.JobTTLSeconds
+		}
+	case schema.StatusJobTerminated, schema.StatusJobFailed:
```
Comment (qiaoshuangshuang) [inline pkg/job/runtime/kubernetes/controller/job_gc_handler.go:81]: pending那个ttl用在哪儿了？

Comment (D0m021ng) [reply pkg/job/runtime/kubernetes/controller/job_gc_handler.go:81]: In job sync, `pendingJobTTLSecond` is used to handle pod in `ImagePullBackOff` or `ErrImagePull`

File: pkg/common/config/config.go (hunk 2 above)
Comment (D0m021ng) [reply pkg/common/config/config.go]: Add prefix `Succeeded` ?

File: pkg/common/config/config.go (hunk 1 above)
Comment (D0m021ng) [reply pkg/common/config/config.go:95]: ok

**Review by qiaoshuangshuang — APPROVED** [review]

**Pull request merged**